### DISCUSSION TITLE
[HW4] Question about problem 3.1

### DISCUSSION THREAD

**Discussion by Hyungi-Lee** [seominjoon/kaist-ai605#160] [Q&A] [answered]
In problem 3.1, it says "You are allowed to copy any code from the **documentation**." and also it says "take a peek at  (but **do not copy!**) https://github.com/huggingface/transformers/tree/master/examples/pytorch/question-answering". So my question is that can we use or copy preprocessing functions and code flows from colab tutorial notebooks provided by hugging face documents?

**Comment by seominjoon** [comment, marked as answer]
I think I was not clear enough here. What I meant is that you should not copy code from the tutorial for question answering, but you can copy code from elsewhere (e.g. sentiment classification).
